specs/sharding/beacon-chain.md

```diff
@@ -103,460 +101,342 @@ The following values are (non-configurable) constants used throughout the specif
+| `SAMPLES_PER_BLOB` | `uint64(2**9)` (= 2,048) | 248 * 2,048 = 507,904 bytes |
 
 ### Precomputed size verification points
 
 | Name | Value |
 | - | - |
 | `G1_SETUP` | Type `List[G1]`. The G1-side trusted setup `[G, G*s, G*s**2....]`; note that the first point is the generator. |
 | `G2_SETUP` | Type `List[G2]`. The G2-side trusted setup `[G, G*s, G*s**2....]` |
-| `ROOT_OF_UNITY` | `pow(PRIMITIVE_ROOT_OF_UNITY, (MODULUS - 1) // int(MAX_SAMPLES_PER_BLOB * POINTS_PER_SAMPLE), MODULUS)` |
-
-### Gwei values
-
-| Name | Value | Unit | Description |
-| - | - | - | - |
-| `MAX_SAMPLE_PRICE` | `Gwei(2**33)` (= 8,589,934,592) | Gwei | Max sample charged for a TARGET-sized shard blob |  
-| `MIN_SAMPLE_PRICE` | `Gwei(2**3)` (= 8) | Gwei | Min sample price charged for a TARGET-sized shard blob |
+| `ROOT_OF_UNITY` | `pow(PRIMITIVE_ROOT_OF_UNITY, (MODULUS - 1) // int(SAMPLES_PER_BLOB * POINTS_PER_SAMPLE), MODULUS)` |
 
 ## Configuration
 
 Note: Some preset variables may become run-time configurable for testnets, but default to a preset while the spec is unstable.  
-E.g. `INITIAL_ACTIVE_SHARDS`, `MAX_SAMPLES_PER_BLOB` and `TARGET_SAMPLES_PER_BLOB`.
+E.g. `ACTIVE_SHARDS` and `SAMPLES_PER_BLOB`.
 
-## Updated containers
-
-The following containers have updated definitions to support Sharding.
+### Time parameters
 
-### `AttestationData`
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `SECONDS_PER_SLOT` | `uint64(8)` | seconds | 8 seconds |
 
-```python
-class AttestationData(Container):
-    slot: Slot
-    index: CommitteeIndex
-    # LMD GHOST vote
-    beacon_block_root: Root
-    # FFG vote
-    source: Checkpoint
-    target: Checkpoint
-    # Hash-tree-root of ShardBlob
-    shard_blob_root: Root  # [New in Sharding]
-```
 
-### `BeaconBlockBody`
+## Updated containers
 
-```python
-class BeaconBlockBody(bellatrix.BeaconBlockBody):  # [extends Bellatrix block body]
-    shard_proposer_slashings: List[ShardProposerSlashing, MAX_SHARD_PROPOSER_SLASHINGS]
-    shard_headers: List[SignedShardBlobHeader, MAX_SHARDS * MAX_SHARD_HEADERS_PER_SHARD]
-```
+The following containers have updated definitions to support Sharding.
 
-### `BeaconState`
+### `IntermediateBlockBid`
 
 ```python
-class BeaconState(bellatrix.BeaconState):
-    # Blob builder registry.
-    blob_builders: List[Builder, BLOB_BUILDER_REGISTRY_LIMIT]
-    blob_builder_balances: List[Gwei, BLOB_BUILDER_REGISTRY_LIMIT]
-    # A ring buffer of the latest slots, with information per active shard.
-    shard_buffer: Vector[List[ShardWork, MAX_SHARDS], SHARD_STATE_MEMORY_SLOTS]
-    shard_sample_price: uint64
-```
+class IntermediateBlockBid(Container):
+    execution_payload_root: Root
 
-## New containers
+    sharded_data_commitment: Root # Root of the sharded data (only data, not beacon/intermediate block commitments)
 
-### `Builder`
+    sharded_data_commitments: uint64 # Count of sharded data commitments
```

Comment:
Rename the first one to `sharded_data_commitment_list_root` and the second `sharded_data_commitment_count` perhaps?